pandas/core/generic.py

```diff
@@ -3092,8 +3092,9 @@ def to_csv(
             this method is called ('\n' for linux, '\r\n' for Windows, i.e.).
 
             .. versionchanged:: 0.24.0
-        chunksize : int or None
+        chunksize : int or None, default None
             Rows to write at a time.
+            If None then it's set to `100000 // (len(self.cols) or 1)) or 1`.
```

Comment:
I think you can remove this line; I'm not sure why this is set this way but its more of an implementation detail than something that needs to be explicitly documented

Rest of the changes look good

Reply:
But that is the default value though, shouldn't that value be explained in the docs?
Since the actual default value is None then only None is shown automatically in the docs.
I had to search through the code to find this implementation because it was affecting my program in an unexpected manner.

I'm sure other packages have this issue as well. There should be a way to define default values like this in a better way or a method that searches the code for the actual default value.
I don't have any experience with this myself, so if there's no chance for another implementation then I'll just remove it like you suggested.

Reply:
Yea let's remove; this is really just an implementation detail and shouldn't mean anything to the end user